dash-info.yaml

```diff
@@ -1,10 +1,2612 @@
+      dontrun: TRUE
+      code: |
+            # Used within htmlColgroup to define columns.
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlTable(list(
+                    htmlColgroup(
+                      list(
+                        htmlCol(span = 2, style = list("background-color"= "red"))
+                      )
+                    ),
+                    htmlTr(
+                      list(
+                        htmlTd("Cell A"),
+                        htmlTd("Cell B"),
+                        htmlTd("Cell C")
+                      )
+                    )
+                  ))
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlColgroup
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlTable(list(
+                    htmlColgroup(
+                      list(
+                        htmlCol(span = 2, style = list("background-color"= "red"))
+                      )
+                    ),
+                    htmlTr(
+                      list(
+                        htmlTd("Cell A"),
+                        htmlTd("Cell B"),
+                        htmlTd("Cell C")
+                      )
+                    )
+                  ))
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlCommand
+      dontrun: TRUE
+      code: |
+            # This component is deprecated and its use is no longer recommended. 
+    - name: htmlContent
+      dontrun: TRUE
+      code: |
+            # This feature is obsolete and no longer supported. It is recommended 
+            # that you use the htmlSlot component instead. 
+    - name: htmlData
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlUl(list(
+                    htmlLi(list(htmlData(value = 398, "First Element"))),
+                    htmlLi(list(htmlData(value = 399, "Second Element"))),
+                    htmlLi(list(htmlData(value = 400, "First Element")))
+                  ))
+                )
+              )
+            )
+            
+            # Include the following in a seperate CSS file in an
+            # `assets` directory in the root of your app.
+            # 
+            # data:hover::after {
+            #   content: ' (ID ' attr(value) ')';
+            #   font-size: .7em;
+            # }
+
+            app$run_server()
+    - name: htmlDatalist
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
```

Comment:
ideally though I suppose we don't want `library(dashCoreComponents)` where it's not used... that happens a lot.

Reply:
I agree, and I'm guilty of the same. It was sloppy to not purge those entries from the examples. We can't always live in the ideal world, but it's better to lead by (good) example.

fixed in 137542f